rasa/core/nlg/interpolator.py

```diff
@@ -38,10 +38,16 @@ def interpolate_text(response: Text, values: Dict[Text, Text]) -> Text:
 
         return text
     except KeyError as e:
+        event_info = (
```

Comment:
These 2 were the only good to have info I could find that we removed from [PR](https://github.com/RasaHQ/rasa/pull/12492/files). Let me know if you think there's more.

Reply:
A few more ideas from me:
- L614 in forms.py:  change the first attribute to `forms.validate.prefilled_slots` or add human readable text
- L129 in facebook.py:  change the first attribute to a negative or add the text
- L324 in hangouts: change to `hangouts.message.blueprint.timeout` or add text
- L171 in rest.py: add `timeout` to first attribute or add text